examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
+
+> If you want to get inspired before moving forward, see the already existing code examples in the folder `examples/next/docs/`.
+
+First off, create a new branch in order to keep the development process clean.
+
+Inside the `examples/templates` directory where all templates for future code examples live. You can see that we have templates for examples made in React, Angular, Vue, and of course vanilla JavaScript.
+
+To create a new code example in Vanilla JavaScript to be used in the Handsontable official documentation, we need to make use of the template from the `examples/templates/js` directory.
+
+#### 1. Copy the existing template
+
+Copy the `examples/templates/js` directory and paste it into `examples/next/docs/js`.
+
+In the next step rename the give that 'js' directory to a name that best describes the purpose of the code example, for instance, "sorting-data-custom-order". Make sure you keep the right file structure, like so:
+
+```
+examples/
+├─ next/
+│  ├─ docs/
+│  │  ├─ js/
+│  │  │  ├─ sorting-data-custom-order/
+```
+
+It's done! Now take a closer look at files inside templates.
 
-1. Add new example using the template within the `/next` directory with paths followed by the convention `/<category>/<framework>/<example_path>` (more info in "Folder structure" section in the [Code Examples](../README.md) README.).
-2. Commit and push the examples to the repo.
-3. In Github Actions run a manual workflow with the `<version_number>` input for the Handsontable and Handsontable's wrapper version. The new folder will be created: `/examples/<version_number>`. Within that folder, new code examples will be created with Handsontable and Handsontable's wrapper version fixed to the `<version_number>`. Note that the Handsontable **version must be available on the NPM** first.
-4. The new code snippets will be deployed to the Github Pages and will be available under URL mentioned in the "Live on production" section in the [Code Examples](../README.md) README.
+#### 2. Template files
 
-### Requirements for a new example
+Every template has files that can be modified to match your needs. But there are some files that you **shouldn't modify** *unless* you know what you are doing (things can break). These files are either framework-specific: `angular.json`, `tsconfig*.json`, `config-overrides.js`, `vue.config.js`, `babel.config.js` or common for every template: `spec/support/jasmine.config.js`.
 
-In order to create new code examples with these templates, each new example must have certain features in the `package.json` file and updated `README.md` with an example's detailed information.
+What you **should modify** though is the `README.md` file. You should add code example-specific content between curly braces. Note: "Live preview" section should have a link to the CodeSandbox, but it won't work until this example is pushed to the repo. You can learn more about CodeSandbox links [here](../#live-on-production).
```

Comment:
have a link to the CodeSandbox,

Don't use the before the brand names if they are not followed by noun. For example:

- Handsontable
- THE Handsontable DATA GRID
- CodeSandbox
- THE CodeSandbox APP
- etc....